src/databricks/labs/ucx/cli.py

```diff
@@ -165,5 +165,48 @@ def revert_migrated_tables(w: WorkspaceClient, schema: str, table: str, *, delet
         tm.revert_migrated_tables(schema, table, delete_managed=delete_managed)
 
 
+@ucx.command
+def migrate_uc_to_uc(
```

Comment:
```suggestion
def move(
```

Repeating for the third time: this command has to be called "move"